### ISSUE TITLE
Make a test_util function for generating a config

### ISSUE THREAD

**Issue by bmw** [certbot/certbot#4434]
All over the place in our tests we create a config object to pass to the code we're testing. We should make a common function/method/class in `certbot.tests.util` that creates a config object with the default values so it can be used throughout our tests.

Related to #4078.

**Comment by nickthetait** [comment]
I'm working on this one

**Comment by nickthetait** [comment]
What sort of default values are you thinking?

**Comment by SwartzCr** [comment]
@bmw that might be a cue to better document this issue

**Comment by bmw** [comment]
I've updated my original post with more information.

**Comment by bmw** [comment]
A version of this defining common values was added in #4974.

**Comment by SwartzCr** [comment]
@bmw should this be closed now that #4974 is merged?

**Comment by bmw** [comment]
I don't think so. That only defined 4 values rather than default values for the entire config. There's still work being done towards this though (see #5037).
